packages/idyll-cli/bin/idyll.js

```diff
@@ -1,6 +1,7 @@
 #! /usr/bin/env node
 
 const idyll = require('../src/');
+const debug = require('debug')('idyll-cli')
```

Comment:
Maybe use `idyll:cli` convention if we want to re-use debug across other modules